scripts/write_output_tables.R

```diff
@@ -10,7 +10,7 @@ if(is.na(params$group_facet)){
 
 for (current_filter in facets) {
   resultsListAll <- overallResListAll[[current_filter]] 
-  resultsListDEGs <- overallResListAll[[current_filter]] 
+  resultsListDEGs <- overallResListDEGs[[current_filter]] # REVIEW: is this right?
```

Comment:
This LOOKS like it was a typo, but would definitely like to double check before merging.

Reply:
Yeah that looks like it was a copy/paste that didn't get fixed